user_tools/src/spark_rapids_pytools/rapids/qualx/prediction.py

```diff
@@ -60,8 +60,15 @@ def _run_rapids_tool(self):
         """
         try:
             output_info = self.__prepare_prediction_output_info()
-            df = predict(platform=self.platform_type.map_to_java_arg(), qual=self.qual_output,
-                         output_info=output_info)
+            estimation_model_args = self.wrapper_options.get('estimationModelArgs')
```

Comment:
Can we simplify this as:
```
estimation_model_args = self.wrapper_options.get('estimationModelArgs', {})
custom_model_file = estimation_model_args.get('customModelFile', None)
```

Reply:
The problem with the suggested form in dictionary that if `wrapper_options['estimationModelArgs'] == None`, then the second statemt will fail because it will be `None.get('customModelFile')`

Reply:
That makes sense. Thanks @amahussein.